<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_reduce</class>
 <widget class="QWidget" name="w_reduce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reduce</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	
	font: 26pt &quot;MS Shell Dlg 2&quot;;
	
	color: rgb(79, 79, 79);
	background-color:  rgba(223, 223, 223, 200);
}


QPushButton
{ 
	alternate-background-color: rgb(255, 255, 255);
	
	background-color: rgb(185, 185, 185);
	border-width: 2px;
    border-radius: 10px;
	position: center;
 }

QPushButton:hover
{
	background: rgba(225, 225, 225, 180);
	border-style: dashed;
}

QPushButton:pressed
{
	
	background-color: rgb(136, 136, 136);
	border-width: 2px;
    border-radius: 10px;
}

QToolButton
{ 
	alternate-background-color: rgb(255, 255, 255);
	
	background-color: rgb(185, 185, 185);
	border-width: 2px;
    border-radius: 10px;
	position: center;
 }

QToolButton:hover
{
	background: rgba(225, 225, 225, 180);
	border-style: dashed;
}

QToolButton:pressed
{
	
	background-color: rgb(136, 136, 136);
	border-width: 2px;
    border-radius: 10px;
}

QLineEdit
{
	background: transparent;
	color: rgb(121, 121, 121);
	border: none;
	border-bottom: 1px solid #717072
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="le_reduction">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>26</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="placeholderText">
      <string>Given amount</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_reduce">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>26</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
